orc8r/cloud/go/services/ctraced/obsidian/handlers/handlers.go

```diff
@@ -255,10 +255,28 @@ func getNetworkIDAndCallTraceID(c echo.Context) (string, string, *echo.HTTPError
 }
 
 func buildStartTraceRequest(cfg *models.CallTraceConfig) (*protos.StartTraceRequest, error) {
-	req := &protos.StartTraceRequest{
-		TraceType: protos.StartTraceRequest_ALL,
+	if cfg.TraceType == models.CallTraceConfigTraceTypeGATEWAY {
```

Comment:
As mentioned elsehere, let's not have separate trace type for gateway and gateway custom